package.json

```diff
@@ -5,6 +5,7 @@
   "scripts": {
     "start": "cross-env MODE=development CONFIG=as-development webpack serve --open",
     "build": "cross-env webpack",
+    "build:dev": "cross-env MODE=development CONFIG=as-development webpack",
```

Comment:
It looks like you figured most of this out, but I'll put my two cents in.

We removed the helpline specific builds so everything can be controlled via env vars. I see you moved this to `build:dev` which won't cause other issues. It also looks like you set up the action build step to set an ENV var like we expect. Is the `build:dev` script still needed?

Reply:
I realized well after that the `build` step was for helpline specific ones / for controlled env vars. I understand it is not needed. 
I got used to the previous scripts for work locally, for purely devx reasons